### ISSUE TITLE
Test suite doesn't work on Windows

### ISSUE THREAD

**Issue by briansmith** [ctz/rustls#16]
```
Output { status: ExitStatus(ExitStatus(101)), stdout: "", stderr: "thread \'main\' panicked at \'cannot open CA file \'/etc/ssl/certs/ca-certificates.crt\': Error { repr: Os { code: 3, message: \"The system cannot find the path specified.\" } }\nConsider using the --cafile option to provide a valid CA file.\', examples\\tlsclient.rs:408\nstack backtrace:\n   0:     0x7ff6a4b7137e - std::panicking::rust_panic_with_hook::h2224f33fb7bf2f4c\n   1:     0x7ff6a4b6fd21 - std::rt::lang_start::h352a66f5026f54bd\n   2:     0x7ff6a4b7075d - std::panicking::rust_panic_with_hook::h2224f33fb7bf2f4c\n   3:     0x7ff6a4b705f6 - std::panicking::begin_panic_fmt::h310416c62f3935b3\n   4:     0x7ff6a4b70504 - std::panicking::begin_panic_fmt::h310416c62f3935b3\n   5:     0x7ff6a4926ff8 - tlsclient::make_config\n                        at C:\\Users\\<...>\\rustls\\<std macros>:8\n   6:     0x7ff6a49279f0 - tlsclient::main\n       
```

Probably rustls should ship with a root CA file that contains the roots that are needed to pass the tests.

**Comment by ctz** [comment]
I have something in the pipeline to fix this and issue #15, along the lines of the certifi group of projects.

**Comment by ctz** [comment]
Should be fixed now as of 0.1.1
